sources/models/src/aws-k8s-1.19/defaults.d/53-containerd-pki.toml

```diff
@@ -0,0 +1 @@
+../../../shared-defaults/containerd-pki.toml
```

Comment:
Nit: I'd prefer "containerd-cri" over "containerd" for clarity in cases where we are doing something that's CRI specific.